apps/ledger-live-desktop/src/renderer/styles/StyleProvider.jsx

```diff
@@ -46,4 +48,17 @@ const StyleProvider = ({ children, selectedPalette }: Props) => {
   );
 };
 
+export const withV2StyleProvider = (Component: React.ComponentType) => {
+  const WrappedComponent = props => {
+    const selectedPalette = useSelector(themeSelector) || "light";
+
+    return (
+      <StyleProvider selectedPalette={selectedPalette}>
+        <Component {...props} />
+      </StyleProvider>
+    );
+  };
+  return WrappedComponent;
+};
+
```

Comment:
It's not part of the scope of this PR, so you don't have to do it. But it looks like this file is easy to transform into a `tsx` file: there is no flow types, and everything is already TS typed. So it could be nice to transform it, to remove all the annoying errors